.upptimerc.yml

```diff
@@ -31,10 +31,9 @@ sites:
     url: https://private.storage/
   - name: Private Storage - payment API
     icon: https://raw.githubusercontent.com/LeastAuthority/end-users-upptime/master/assets/private-storage-icon.svg
-    url: https://payments.private.storage/v1/redeem
-    method: POST
```

Comment:
Please don't do this. A "post" is always meant to create or modify state on the server.  This is not a well-suited liveliness-check endpoint (and screws over our statistics).